app/models/setting/provisioning.rb

```diff
@@ -23,9 +23,18 @@ def self.load_defaults
         self.set('use_shortname_for_vms', N_("Foreman will use the short hostname instead of the FQDN for creating new virtual machines"), false, N_('Use short name for VMs')),
         self.set('dns_conflict_timeout', N_("Timeout for DNS conflict validation (in seconds)"), 3, N_('DNS conflict timeout')),
         self.set('clean_up_failed_deployment', N_("Foreman will delete virtual machine if provisioning script ends with non zero exit code"), true, N_('Clean up failed deployment')),
+        self.set('name_generator_type', N_("Type of name generator"), 'Off', N_("Name generator type")),
+        self.set('name_generator_seed', N_("Initial LFSR seed value (do not change)"), -1, N_("Name generator seed")),
```

Comment:
:+1:  - I'd remove all options but the 'type' one, and change that to just  A few reasons:

* It's complicated (LFSR? most users won't know what it is nor they should need to know how the sausage is being made)
* User is given zero information why would they change that - I can't think of a good reason myself either
* It's yet another source of bugs